### DISCUSSION TITLE
Are there any options to always render a custom 404?

### DISCUSSION THREAD

**Discussion by Mifaresss** [amannn/next-intl#874] [Q&A]
### Is your feature request related to a problem? Please describe.

Why I should render default 404 page if pathname contains "."? I really don't understand...Is it difficult to introduce another variant?

### Describe the solution you'd like

I wan't this url https://next-intl-example-app-router.vercel.app/somepage.asdf also return custom NotFound

### Describe alternatives you've considered

...

**Comment by amannn** [comment]
You can absolutely do so in your app (see [`not-found.tsx`](https://github.com/amannn/next-intl/blob/main/examples/example-app-router/src/app/not-found.tsx)), but note that you don't have a `locale` in this route—therefore either you have to assume one or redirect to a prefixed route.

Moving this to a discussion since it's a usage question …
